Essentials/src/main/java/com/earth2me/essentials/commands/Commandafk.java

```diff
@@ -50,55 +49,63 @@ private void toggleAfk(final User sender, final User user, final String message)
             if (sender.isMuted()) {
                 final String dateDiff = sender.getMuteTimeout() > 0 ? DateUtil.formatDateDiff(sender.getMuteTimeout()) : null;
                 if (dateDiff == null) {
-                    throw new Exception(sender.hasMuteReason() ? tl("voiceSilencedReason", sender.getMuteReason()) : tl("voiceSilenced"));
+                    if (sender.hasMuteReason()) {
+                        throw new TranslatableException("voiceSilencedReason", sender.getMuteReason());
+                    } else {
+                        throw new TranslatableException("voiceSilenced");
+                    }
+                }
+                if (sender.hasMuteReason()) {
+                    throw new TranslatableException("voiceSilencedReasonTime", dateDiff, sender.getMuteReason());
+                } else {
+                    throw new TranslatableException("voiceSilencedTime", dateDiff);
                 }
-                throw new Exception(sender.hasMuteReason() ? tl("voiceSilencedReasonTime", dateDiff, sender.getMuteReason()) : tl("voiceSilencedTime", dateDiff));
             }
```

Comment:
This code has been copied around a few places, would it be worth pulling out into something like `User::getMutedException`?